### ISSUE TITLE
R: Fix hover help for function arguments

### ISSUE THREAD

**Issue by juliasilge** [posit-dev/positron#5302]
_Originally posted by @qpmnguyen in https://github.com/posit-dev/positron/discussions/3773_:

Hi all, 

I love working with Positron so far and it would be great if some of these features would be added in the future (if not already planned): 

First, adding the ability to pull in a summary of the functional argument documentation. For example in RStudio:
![image](https://github.com/posit-dev/positron/assets/25627274/80b4214d-5428-4556-af51-c150d360f432)

**Comment by DavisVaughan** [comment]
Note that this is (i think) the same as https://github.com/posit-dev/positron/issues/4046 and used to work. Since this issue has more 👍 im going to pull over the details from there:

---


With R 4.2

<img width="607" alt="Screenshot 2024-07-17 at 9 14 37 AM" src="https://github.com/user-attachments/assets/d862b4c7-2df0-499b-b48f-63fe4cd3e3db">

With R 4.3.1

<img width="349" alt="Screenshot 2024-07-17 at 9 14 53 AM" src="https://github.com/user-attachments/assets/b9459e36-2178-46d3-bc8c-fb2c245d6908">

The problem is similar to https://github.com/posit-dev/ark/pull/119

At this point:
https://github.com/posit-dev/ark/blob/47d6ee885cc1827a3683eaa1d9eaabb94933eab8/crates/ark/src/lsp/help.rs#L163

we are looking for a selector of `tr[style="vertical-align: top;"] > td`. As of R 4.3.1, that `vertical-align: top` style is no longer hardcoded into the HTML. It was moved to `R.css` (which we think is good in the long run). That breaks our selector, and it no longer comes up with any selections, so we don't get any per argument help info.

**Comment by olivroy** [comment]
Will this be supported soon? I know this is marked as "Future", but the original issue had "Release Candidate". Since this works on previous versions of R, it would be great if it worked for R 4.3, 4.4

**Comment by juliasilge** [comment]
Yes, let me move this to release candidate. We do in fact have this one in a list to do before coming out of beta, so it should be in that milestone. Thanks!

**Comment by olivroy** [comment]
Works perfectly in the 2025.05 pre-release + R 4.4.1. Thanks for fixing

**Comment by midleman** [comment]
### Verified Fixed
Positron Version(s) : [2025.05.0-103](https://github.com/posit-dev/positron-builds/releases/tag/2025.05.0-103)
OS Version(s) : MacOS

### Test scenario(s) 
Confirmed hover help appears as expected for R 4.3.2, and 4.4.1

### Link(s) to test cases run or created: 
n/a
